config/manager/manager.yaml

```diff
@@ -62,6 +62,26 @@ spec:
       protocol: TCP
       port: 8888
       targetPort: 8888
+    - name: metrics
+      protocol: TCP
+      port: 8080
+      targetPort: 8080
   selector:
     control-plane: controller-manager
   type: ClusterIP
+---
+kind: ServiceMonitor
+apiVersion: monitoring.coreos.com/v1
+metadata:
+  name: k4e-operator-servicemonitor
+  namespace: system
```

Comment:
k4e-operator-system

Reply:
iiuc, kustomize is responsible to replace it with the actual namespace in the created resource: https://github.com/jakub-dzon/k4e-operator/blob/main/config/default/kustomization.yaml#L2